scene/animation/animation_player.cpp

```diff
@@ -199,816 +131,35 @@ void AnimationPlayer::_validate_property(PropertyInfo &p_property) const {
 void AnimationPlayer::_get_property_list(List<PropertyInfo> *p_list) const {
 	List<PropertyInfo> anim_names;
 
-	anim_names.push_back(PropertyInfo(Variant::DICTIONARY, PNAME("libraries")));
-
 	for (const KeyValue<StringName, AnimationData> &E : animation_set) {
-		if (E.value.next != StringName()) {
+		if (animation_next_set.has(E.key) && animation_next_set[E.key] != StringName()) {
```

Comment:
```suggestion
		HashMap<StringName, StringName>::ConstIterator F = animation_next_set.find(E.key);
		if (F && F->value != StringName()) {
```
Avoids double lookup.